access_card/src/main/java/edu/artic/accesscard/barcode/BarcodeEncoder.kt

```diff
@@ -0,0 +1,78 @@
+package edu.artic.accesscard.barcode
+
+
+import android.graphics.Bitmap
+
+import com.google.zxing.BarcodeFormat
+import com.google.zxing.EncodeHintType
+import com.google.zxing.MultiFormatWriter
+import com.google.zxing.WriterException
+import com.google.zxing.common.BitMatrix
+
+
+/**
+ * Helper class for encoding barcodes as a Bitmap.
+ *
+ * Adapted from QRCodeEncoder, from the zxing project:
+ * https://github.com/zxing/zxing
+ *
+ * Licensed under the Apache License, Version 2.0.
```

Comment:
I think we might need to add a more explicit license statement here. Not sure. I'll bring it up with legal.